File: JavaScript/1008.js
var input = require('fs').readFileSync('stdin', 'utf8');

var valores = input.split("\n");
var Numf = parseInt(valores.shift());
var Numh = parseInt(valores.shift());
var Numv = parseFloat(valores.shift());

console.log("NUMBER = " + Numf + "\n" + "SALARY = U$ " + (Numh*Numv).toFixed(2));

File: C/1008.c
#include <stdio.h>
#include <stdlib.h>

int main() {

    int NUMBER;
    int w;
    float SALARY;
    float p;
    scanf("%d", &NUMBER);
    scanf("%d", &w);
    scanf("%f", &p);
    SALARY = w * p;
    printf("NUMBER = %d\n", NUMBER);
    printf("SALARY = U$ %.2f\n", SALARY);

    return 0;
}